sqlobject/src/main/java/org/jdbi/v3/sqlobject/SqlObjectFactory.java

```diff
@@ -67,7 +68,9 @@ public boolean accepts(Class<?> extensionType) {
 
         return Stream.of(extensionType.getMethods())
                 .flatMap(m -> Stream.of(m.getAnnotations()))
-                .anyMatch(a -> a.annotationType().isAnnotationPresent(SqlMethodAnnotation.class));
+                .map(a -> a.annotationType())
+                .anyMatch(type -> type.isAnnotationPresent(SqlMethodAnnotation.class) ||
+                        type.isAnnotationPresent(SqlMethodDecoratingAnnotation.class));
```

Comment:
This check for `SqlMethodDecoratingAnnotation` might not be necessary. I'm still thinking it through, but I believe that testing for mixins interfaces or `SqlMethodAnnotation` should be sufficient.